frontend/src/app/testResults/TestResultsList.test.tsx

```diff
@@ -664,24 +664,6 @@ describe("TestResultsList", () => {
       userEvent.selectOptions(screen.getByLabelText("Testing facility"), ["3"]);
       expect((await screen.findAllByText("Flu A"))[0]).toBeInTheDocument();
     });
-
-    it("should default to column value and show N/A for Flu results for rows with no multiplex results", async () => {
```

Comment:
⚠️ removed this test since it is no longer applicable

Reply:
I'm confused about this, it does still show N/A for rows with no multiplex results. Maybe I'm misunderstanding the test, but don't we want to still ensure it renders this way?

Reply:
Or is it that we just want to test it in a different location? e.g. the ResultsTable.test.tsx?

Reply:
Actually nvm, I see it was moved to the other test so with the restructuring it makes sense! don't mind me lol